pytest_tests/lib/helpers/grpc_responses.py

```diff
@@ -40,6 +40,8 @@
     "table validation: it is prohibited to modify system access"
 )
 
+INVALID_SEARCH_QUERY = ".*invalid search query.*"
```

Comment:
i'd also add query(s) with floating point numbers which are also currently unsupported. `attr<1.2` is incorrect as well as `attr<text` but not so obvious

Reply:
added this case as well